src/views/List.jsx

```diff
@@ -11,29 +12,35 @@ export function List({ data }) {
 
 	return (
 		<>
-			<form className="filterForm">
-				<label htmlFor="searchItems">Filter Items</label>
-				<input
-					type="text"
-					id="searchItems"
-					name="searchItems"
-					value={searchQuery}
-					placeholder="Start typing here"
-					autoComplete="off"
-					onChange={(e) => setSearchQuery(e.target.value)}
-				/>
-				{searchQuery && <button onClick={clearInput}>Clear</button>}
-			</form>
-
-			<ul>
-				{data
-					.filter((item) =>
-						item.name.toLowerCase().includes(searchQuery.toLowerCase()),
-					)
-					.map((item) => {
-						return <ListItem key={item.id} name={item.name} />;
-					})}
-			</ul>
+			{data.length > 0 ? (
+				<ul>
```

Comment:
Curious about the reasoning for moving this `<ul>` from coming after the `<form>` to now wrapping the `<form>`. I might be missing something, but it seems like it makes sense to keep it after the `<form>`.

Reply:
We overlooked that. I updated it